### DISCUSSION TITLE
Need help on how to set up temperature probes for pool and spa

### DISCUSSION THREAD

**Discussion by dkossman** [tagyoureit/nodejs-poolController#1029] [REM]
- I have a single-body pool/spa setup
- i have a pentair mastertemp heater and a pentair VS pump connected via RS485 
- i have a 10k temp probe in the return line as advised previously
- I have 2 circuits set up, a pool circuit that runs the pump at 1000rpm and a spa circuit that runs the pump at 2500 rpm
- I also have 3 DS18B20 probes: one in the deep end of the pool, one in the spa, and one for air temp
- all 4 probes are working in REM and reporting correct temperatures

I'm confused about how to set up my temperature probes with REM and NJSPC.  things are not working as expected...

- I'm feeding the air temp probe to NJSPC [temps].airSensor.  this seems to work, i see the correct air temp in dashpanel
- I'm feeding the 10k return line probe to NJSPC [temps].waterSensor1.  this does not work, i don't see the correct temperature in dashpanel when the pump is running.  i'm not sure where it is getting the temperature from.  my DS18B20 pool temp probe is not active in REM
- i'm feeding the DS18B20 Spa temperature to [temps].waterSensor2 (not sure if this is correct?)

-> what are [temps].waterSensor1 , [temps].waterSensor2, etc?
- does the RS485 mastertemp pump interface read the heater's temperature sensor (as reported on the heater's panel)?
- if i'm in spa mode, should the system be reading the temperature from the return line probe, or the probe in the spa?  or from the heater itself?  
- i assume that when i'm in pool mode, the system should read the temperature from the return line probe.  or does it read it from the heater itself?
- is there some other setup i need to do in REM so that the pool and spa circuits are mapped to the right temperature sensors?

thanks for any help from anyone...

**Comment by dkossman** [comment]
OK - mostly figured this out.  in trying to get things working I had at some point deleted my ADS1115 in REM :-(.  I added it back and configured it to send data to the internal 10k temperature probe and things are working.   sorry for anyone's wasted time.

so it appears that in both Spa and Pool modes, the water temp reported on dashpanel is [temps].waterSensor1?   but what are waterSensor2, 3, and 4?  

thanks

**Reply by tagyoureit** [reply]
waterSensor2 is for dual bodies.  3 & 4 aren't really used.

**Comment by dkossman** [comment]
thanks.  closing this out.
